debian/control

```diff
@@ -0,0 +1,34 @@
+Source: ubuntu-insights
+Section: golang
+Priority: optional
+Maintainer: Ubuntu Developers <ubuntu-devel-discuss@lists.ubuntu.com>
+Rules-Requires-Root: no
+Build-Depends: debhelper-compat (= 13),
+               dh-golang,
+               golang-1.24-go,
+Testsuite: autopkgtest-pkg-go
+Standards-Version: 4.6.2
+Vcs-Browser: https://salsa.debian.org/go-team/packages/ubuntu-insights
+Vcs-Git: https://salsa.debian.org/go-team/packages/ubuntu-insights.git
+Homepage: https://github.com/ubuntu/ubuntu-insights
+XS-Go-Import-Path: github.com/ubuntu/ubuntu-insights
+
+Package: ubuntu-insights
+Architecture: any
+Depends: ${misc:Depends},
+         ${shlibs:Depends}
+Built-Using: ${misc:Built-Using}
+Description: Ubuntu Insights (program)
+ Ubuntu Insights, a user transparent, open, 
+ platform-agnostic and cross application solution for reporting
+ hardware information and other collected metrics.
+ .
+ Ubuntu Insight is designed to show you exactly what is being sent, and
+ allow you to acknowledge and control your own data. The information 
+ can't be used to identify a single machine, and is cached for local viewing
+ before being sent to the server.
+ .
+ This is designed to be a full replacement for Ubuntu Report.
+ .
+ This package contains the command line tool.
```

Comment:
is it only a command line tool, because we are shipping the systemd services trigger too, no?